octobot_commons/databases/run_databases/run_databases_pruner.py

```diff
@@ -0,0 +1,195 @@
+#  Drakkar-Software OctoBot-Trading
+#  Copyright (c) Drakkar-Software, All rights reserved.
+#
+#  This library is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU Lesser General Public
+#  License as published by the Free Software Foundation; either
+#  version 3.0 of the License, or (at your option) any later version.
+#
+#  This library is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with this library.
+import os
+import shutil
+import time
+import pathlib
+
+import octobot_commons.enums as enums
+import octobot_commons.constants as constants
+import octobot_commons.logging as logging
+import octobot_commons.errors as errors
+import octobot_commons.databases as databases
+
+
+class RunDatabasesPruner:
+    def __init__(self, databases_root_path, max_databases_size, database_adaptor):
+        self.logger = logging.get_logger(self.__class__.__name__)
+        self.databases_root_path = databases_root_path
+        self.max_databases_size = max_databases_size
+        self.database_adaptor = database_adaptor
+        self.all_db_data = []
+
+        separator = os.path.sep if self.database_adaptor.is_file_system_based else constants.DB_SEPARATOR
+        self.backtesting_run_path_identifier = {
+            f"{separator}{enums.RunDatabases.BACKTESTING.value}{separator}",
+            f"{separator}{enums.RunDatabases.OPTIMIZER.value}{separator}"
+        }
+        self._run_db = f"{enums.RunDatabases.RUN_DATA_DB.value}{self.database_adaptor.get_db_file_ext()}"
```

Comment:
Do we have some utility methods somewhere to do this? If not, I think we should create them.